packages/fiori-generator-shared/src/helpers.ts

```diff
@@ -58,3 +58,15 @@ export function getDefaultTargetFolder(vscode: any): string | undefined {
     // Otherwise use <home-dir>/projects,
     return existsSync(DEFAULT_PROJECTS_FOLDER) ? DEFAULT_PROJECTS_FOLDER : undefined;
 }
+
+/**
+ * Removes any `-api` suffix in the first label of the hostname.
```

Comment:
You might add the `why` we are adding this method? For example, an external system knows it should connect to https://<tenant-name>-api.s4hana.ondemand.com for API calls, as opposed to interacting with the standard UI hostname, https://<tenant-name>.s4hana.ondemand.com.

Reply:
Comment added